auto-notify.plugin.zsh

```diff
@@ -54,7 +54,7 @@ function _auto_notify_message() {
         local urgency="normal"
         local transient="--hint=int:transient:1"
         local icon=${AUTO_NOTIFY_ICON_SUCCESS:-""}
-        if [[ "$exit_code" != "0" ]]; then
+        if [[ "$exit_code" != "0" ]] && [[ "$exit_code" != "130" ]]; then
```

Comment:
could you add a comment explaining the reasoning behind ignoring 130?

Essentially a short form of what you put in the pull request description :)